### PR TITLE
Eq 2174 lms using proxy titles

### PR DESCRIPTION
**Pull request by AndrewTorrance**
Modified LMS_1.Json to use titles instead of duplicating two arms of the survey. In doing so it was noticed that dumping did not work . This was fixed with a change to dump.py. This is known not to be ideal ( setting a global for use lower down) However it has been agreed to look at the use of globals as a separate ticket.
(Note this replaces an earlier PR where I messed up a merge.)

### How to review 
Run lms_1 with multiple users and verify functionality. Check submission works prior to submitting

### CODE REVIEW

**Review by kierenbeckett — APPROVED** [review]

File: app/views/dump.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ def dump_submission():
     answer_store = get_answer_store(current_user)
     metadata = get_metadata(current_user)
     session_data = get_session_store().session_data
-    schema = load_schema_from_session_data(session_data)
+    g.schema = load_schema_from_session_data(session_data)
```
Comment (jonnyshaw89) [inline app/views/dump.py]: why?

Comment (jonnyshaw89) [inline app/views/dump.py]: We should make this a separate PR as its unrelated to the change

Comment (AndrewTorrance) [reply app/views/dump.py]: Agree, changing it

**Review by nealedj — APPROVED** [review]

**Comment by nealedj** [comment]
LGTM - @AndrewTorrance could you squash those 3 commits and rebase this branch on master?
